tgui/packages/tgui/interfaces/ChemiCompiler/ChemiCompilerMemory.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author Garash2k (https://github.com/Garash2k)
+ * @license ISC
+ */
+
+import { useBackend } from '../../backend';
+import { ChemiCompilerData } from './type';
+import { Button, Section, Stack } from '../../components';
+
+export const ChemiCompilerMemory = (_props, context) => {
+  const { act, data } = useBackend<ChemiCompilerData>(context);
+  const { buttons, sx, tx, ax, theme } = data;
+  return (
+    <Section title="Memory">
+      sx: <em>{sx}</em> -
+      ax: <em>{tx}</em> -
+      tx: <em>{ax}</em>
```

Comment:
All of this review comes with the caveat of "I don't know the ChemiCompiler well", but:

Flat text label and italic values here are bleh, given the screenshot context. Bare minimum consider putting these in their own `Stack` with `grow` applied to the items so they fill the space. `-` is not a good "spacer" these days.